module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  POST, GET, PUT, DELETE
+
 2. What is Sinatra?
+  A lightweight version of Ruby on Rails.
+  
 4. What is MVC?
+  Model View Controller
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  Consistency throughout the code.  Makes for easier debugging/troubleshooting.
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Instance, Local, and Class variables.

```

Comment:
the only variable that's accessible in our view template without explicitly passing it is an instance variable